Data/DataModel/Event.cs

```diff
@@ -29,19 +29,70 @@ public Uri URL
         public bool IsInternEvent { get; set; }
         public DateTime TeamRegistrationBegin { get; set; }
         public DateTime TeamRegistrationEnd { get; set; }
+
+        /// <summary>
+        /// Returns whether or not team registration is active.
+        /// </summary>
+        /// <returns>True if the current date is between the team registration begin and end times.</returns>
+        public bool IsTeamRegistrationActive()
+        {
+            return DateTime.Now.CompareTo(TeamRegistrationBegin) > 0 &&
+                DateTime.Now.CompareTo(TeamRegistrationEnd) < 0;
+        }
+
         public DateTime TeamNameChangeEnd { get; set; }
         public DateTime TeamMembershipChangeEnd { get; set; }
         public DateTime TeamMiscDataChangeEnd { get; set; }
         public DateTime TeamDeleteEnd { get; set; }
         public DateTime EventBegin { get; set; }
+
+        /// <summary>
+        /// Returns whether or not the event has started. Does not necessarily indicate the event
+        /// is currently active.
+        /// </summary>
+        /// <returns>True if the current date is after the EventBegin time</returns>
+        public bool IsEventStarted()
```

Comment:
nit: EventHasStarted
How is "active" defined for an event if not based on the start time?

Reply:
This only compares the current time to the event begin time. So, after the event has started, this will return true. When the event ends, the event is not 'active' but this will continue to return true.